TSQL_Parser/TSQL_Parser/Clauses/Parsers/TSQLOffsetClauseParser.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using TSQL.Clauses;
+using TSQL.Clauses.Parsers;
+using TSQL.Tokens;
+
+namespace TSQL.Statements.Parsers
+{
+    internal class TSQLOffsetClauseParser
+    {
+        public TSQLOffesetClause Parse(ITSQLTokenizer tokenizer)
+        {
+            TSQLOffesetClause offset = new TSQLOffesetClause();
+
+            if (!tokenizer.Current.IsKeyword(TSQLKeywords.OFFSET))
+            {
+                throw new InvalidOperationException("OFFSET expected.");
+            }
+
+            offset.Tokens.Add(tokenizer.Current);
+            
+            TSQLSubqueryHelper.ReadUntilStop(
+                tokenizer,
+                offset,
+                new List<TSQLFutureKeywords>
+                {
+                    Capacity = 0
```

Comment:
Explicitly setting the Capacity seems odd, so I figured that this was probably an interesting optimization trick that you knew, but it appears that 0 is the default value for Capacity, so this line is unnecessary, so I'd recommend removing it so no one gets the impression that we're doing something special here.

Reply:
I think this was just in another parser that I copied and pasted as the starting point.  Didn't really intend to set a capacity.